content/response-to-reviewers.md

```diff
@@ -179,6 +185,13 @@ e.g. I see a lot of utility in manubot-style approaches for Carpentries-style on
 I assume this happens but want to double check (and am at the far end of a lousy Internet connection right now).
 So this is really more of an "@ctb todo" item.
 
+We discuss this comment further in [GH121](https://github.com/greenelab/meta-review/issues/121).
+Rendered versions of manuscripts do not currently get uploaded to pull requests.
+Instead, pull requests must be built locally to view the user-facing outputs (i.e. HTML or PDF).
+This is a [limitation](https://github.com/travis-ci/travis-ci/issues/5579) of pull request builds on Travis CI, which are unable to [upload artifacts](https://docs.travis-ci.com/user/uploading-artifacts/) due to security restriction.
+Given the importance of exposing rendered manuscripts from pull requests, we will closely monitor for possible solutions.
```

Comment:
I suggest here or in the next line we link to a new issue in `manubot/rootstock` to show that we are actively working on this as a high priority issue.  Getting this to work would make reviewing pull requests significantly easier.